model/src/commonMain/kotlin/io/github/droidkaigi/feeder/TimetableContents.kt

```diff
@@ -7,7 +7,23 @@ import kotlinx.datetime.toInstant
 data class TimetableContents(
     val timetableItems: TimetableItemList = TimetableItemList(),
     val favorites: Set<String> = setOf(),
-)
+) {
+    val contents by lazy {
```

Comment:
I think it would be simpler to make TimetableItemList hold a map, but since it affects iOS as well, let's go with this policy for now. 👍